src/main/java/de/hpi/bpt/argos/api/response/ResponseFactoryImpl.java

```diff
@@ -25,15 +33,21 @@
 public class ResponseFactoryImpl implements ResponseFactory {
 	protected static final Logger logger = LoggerFactory.getLogger(ResponseFactoryImpl.class);
 	protected static final Gson serializer = new Gson();
+	protected static final JsonParser jsonParser = new JsonParser();
+
+	protected static final String JSON_EVENT_QUERY_ATTRIBUTE = "eventQuery";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "eventQuery" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1192)